src/base/bittorrent/session.cpp

```diff
@@ -2895,10 +2895,10 @@ QString Session::networkInterface() const
     return m_networkInterface;
 }
 
-void Session::setNetworkInterface(const QString &iface)
+void Session::setNetworkInterface(const QString &interface)
```

Comment:
IMO, "iface" is well-known shorthand so it's acceptable.

Reply:
I can change them both to `iface`. The problem was that the naming was inconsistent between declaration and definition.

Reply:
>The problem was that the naming was inconsistent between declaration and definition.

Oh, I didn't pay attention. Then whatever you want...